fmn/consumer/backend.py

```diff
@@ -231,10 +231,10 @@ def read(queue_object):
 lc3 = task.LoopingCall(confirmation_producer.work)
 lc3.start(frequency)
 
-digest_producer = fmn_producers.DigestProducer(
-    session, backends)
-lc4 = task.LoopingCall(digest_producer.work)
-lc4.start(frequency)
+#digest_producer = fmn_producers.DigestProducer(
+#    session, backends)
+#lc4 = task.LoopingCall(digest_producer.work)
+#lc4.start(frequency)
```

Comment:
If you want to drop this code, let's just delete it - if we need it back one day it'll be in the git history.